### PR TITLE
[Wiki] Add further details to identity bug of RSV to documentation

### PR DESCRIPTION
**Pull request by fblix**
# Description

This PR adds further details of the ongoing identity issues with the Recovery Services Vault.
Refer to #2391 

# Type of Change

- [x] Update to documentation

# Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

cc: @JPEasier

### CODE REVIEW

**Comment by eriqua** [comment]
> @eriqua this seems to be a bigger issue related to Azure (either RSV PG or Bicep). How should we update your issue ticket (#2391) since the issue does not seem to be resolvable by us alone.
> 
> See [issue in the bicep repo](https://github.com/Azure/bicep/issues/9662) for more details.

@fblix thanks for testing and opening the bug! To your question: replied directly in issue #2391

File: docs/wiki/Known issues.md [hunk 1]
```diff
@@ -45,9 +45,17 @@ Further details are also provided in issue [#1342](https://github.com/Azure/Reso
...
+The Recovery Services Vaults module does not currently attach the content of the identity property correctly when both user- and systemassigned identity fields are selected.
 
-The module pipeline fails in the deployment validation step when system and user assigned identity parameters are added as input parameters.
+The pipeline shows a success but the assignment of both identities never happens although both identities (systemassigned is a serviceprincipal, userassigned is a managed identity resource) get created successfully.
+
+Upon cleanup the system-assigned identity will not be removed.
+
+When the deployment is then run again it fails, because Azure tries to attach this rogue service principal as a system-assigned identity.
+
+The branch `users/fblix_jpeasier/rsvIdentity` has been used to testing and contains failing and succeeding samples.
```
Comment (eriqua) [inline docs/wiki/Known issues.md:56]: I'd suggest to remove this paragraph. The referenced dev branch will likely get stale going forward.

File: docs/wiki/Known issues.md [hunk 2]
```diff
@@ -45,9 +45,17 @@ Further details are also provided in issue [#1342](https://github.com/Azure/Reso
...
+
+Since the behavior is inconsistent via Api (depending on spacing and whether capital letters are used), a ticket on the bicep repository has been opened for that. For more details, refer to the issue in the bicep repository ([#9662](https://github.com/Azure/bicep/issues/9662)).
```
Comment (eriqua) [inline docs/wiki/Known issues.md:58]:
```suggestion
For more details, refer to the issue in the Bicep repository ([#9662](https://github.com/Azure/bicep/issues/9662)).
```

File: docs/wiki/Known issues.md [hunk 3]
```diff
@@ -45,9 +45,17 @@ Further details are also provided in issue [#1342](https://github.com/Azure/Reso
...
 
 A related issue has been opened in the Bug board [#2391](https://github.com/Azure/ResourceModules/issues/2391).
```
Comment (eriqua) [inline docs/wiki/Known issues.md:60]: If the above suggestion gets accepted, please remove this paragraph

**Review by eriqua — COMMENTED** [review]
Hi @fblix! Thanks again for your contribution and for raising the issue to the PG. Please let me know if suggested changes make sense to you or if we are waiting for anything in the main issue to be addressed before merging this in.

**Review by JPEasier — DISMISSED** [review]

File: docs/wiki/Known issues.md (hunk 1 above)
Comment (fblix) [reply docs/wiki/Known issues.md]: agreed!

File: docs/wiki/Known issues.md (hunk 3 above)
Comment (fblix) [reply docs/wiki/Known issues.md:58]: done

**Pull request merged**